### ISSUE TITLE
Support for Cheetah

### ISSUE THREAD

**Issue by adoublef** [denoland/deno_kv_oauth#146]
I came across a new Deno framework and was attempting to use the package for auth. seems I am having issues with a response. The author says that the `ctx.req.raw` object is a standard `Request` and that did work for the signIn leg, but for the callback seems to just not do anything. I am looking into this

issue: [add support for denoland/deno\_kv\_oauth · Issue #133 · azurystudio/cheetah (github.com)](https://github.com/azurystudio/cheetah/issues/133)

**Comment by adoublef** [comment]
Working on a mini demo as I have been able to overcome some of the API differences. Cheetah is new and looks to have a fair amount of good features included jsx support so may be good to add an example for it?

**Comment by iuioiua** [comment]
A Cheetah example would be welcome!

**Comment by adoublef** [comment]
yes, I can do that for sure, where would examples be best to add?

**Comment by iuioiua** [comment]
If Cheetah can be written in one file, I'd suggest creating a Deno Deploy playground and linking to that, like the [Oak example](https://github.com/denoland/deno_kv_oauth#in-the-wild). If it requires its own repo, we can link to that like the other examples. Either way, please keep it as minimal as possible and the front end essentially identical to [the main demo](https://github.com/denoland/deno_kv_oauth/blob/97eeea8640b4b5b558dc8e9adfa9b28ec6c20749/demo.ts).

**Comment by iuioiua** [comment]
Hey @adoublef, just checking in. Did you manage to create a Cheetah demo?

**Comment by adoublef** [comment]
Hello, I can confirm I've got it working now. I've essentially used the template for the Oak demo to test with Auth0 as it works well. 

A bit unsure what I missed in the initial issue, but can confirm I'm able to get the response back from the callback

**Comment by iuioiua** [comment]
Nice! Please feel free to open a PR that links to your example in the "In the Wild" section of the README 😄
